netbout-web/src/main/java/com/netbout/dynamo/DyAlias.java

```diff
@@ -70,15 +81,6 @@
      */
     private final transient Item item;
 
-    static {
```

Comment:
@mkordas  ``` static{}``` is not used anymore I did the initialization in another way to prevent the violation in with the final field .